src/net.cpp

```diff
@@ -2813,35 +2813,37 @@ bool CConnman::NodeFullyConnected(const CNode* pnode)
 
 void CConnman::PushMessage(CNode* pnode, CSerializedNetMsg&& msg)
 {
-    size_t nMessageSize = msg.data.size();
-    LogPrint(BCLog::NET, "sending %s (%d bytes) peer=%d\n",  SanitizeString(msg.m_type), nMessageSize, pnode->GetId());
+    size_t message_size = msg.data.size();
```

Comment:
d093d261e2df27b2466ffe4e92bd48f22818d326
```suggestion
    const size_t message_size = msg.data.size();
```

but I think that the code would look better without the `message_size` variable at all.

Reply:
I don't have a strong opinion on the removal of `message_size` and/or `total_size`.  However such a change is a bit more than I'm looking to do here.